XCUITests/PrivateBrowsingTest.swift

```diff
@@ -4,11 +4,12 @@
 
 import XCTest
 
-let url1 = "www.mozilla.org"
-let url2 = "www.facebook.com"
+let url1 = "example.com"
+let url2 = path(forTestPage: "test-mozilla-org.html")
+let url3 = path(forTestPage: "test-example.html")
 
-let url1Label = "Internet for people, not profit — Mozilla"
-let url2Label = "Facebook - Log In or Sign Up"
+let url1And3Label = "Example Domain"
```

Comment:
@isabelrios correct me if I'm wrong but my feeling is that this should be "url1Label" not "url1And3Label". Might be a typo.

Reply:
@SimonBasca it is not a typo :) the label is the same for url1 and url3, Example Domani, since the url is the same (one is the actual website and the other the same but served locally) and so I tried to use only one variable for both urls. Obviously it is not clear :) let me think a better naming...

Reply:
I get it now. It's not something too confusing, we can leave it like it is.